src/scss/_dependencies.scss

```diff
@@ -26,7 +26,7 @@
 @import '@justeat/pie-design-tokens/dist/jet';
 
 @import 'settings/variables';
-@import '@justeat/f-utils/src/scss/helpers/breakpoints'; // breakpoint helper – https://github.com/justeat/f-utils/blob/96508ae19809c4b8b83cebb55b9d74728d01c1a5/src/scss/helpers/_breakpoints.scss
+@import 'tools/helpers/breakpoints'; // breakpoint helper
```

Comment:
Would be good to remove this by default and add it into the `_templates` as I think we only really use this on some of our older applications.

Would make this change a breaking change then though, so maybe could you add a `TODO` to this line that we'll remove/move it as part of the next major version bump?